src/Lykke.Logs/Console/LykkeConsoleLoggerProvider.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Console;
+using Microsoft.Extensions.Options;
+
+namespace Lykke.Logs
+{
+    [ProviderAlias("Console")]
+    public sealed class LykkeConsoleLoggerProvider : ILoggerProvider
```

Comment:
Add XML Docs for public class, or it could be made internal?

Reply:
ok